app/src/main/res/values/strings.xml

```diff
@@ -8,14 +8,14 @@
     <string name="install">Install</string>
     <string name="cancel">Cancel</string>
     <string name="fdroid_vlc_url" translatable="false">https://f-droid.org/repository/browse/?fdfilter=vlc&amp;fdid=org.videolan.vlc</string>
-    <string name="open_in_browser">Open in browser</string>
+    <string name="open_in_browser">Open in web-browser</string>
```

Comment:
?

Reply:
We should keep the old version, because it is shorter and there should be any room for confusion with a file browser.

Reply:
@TobiGr The problem that arises when there "is any room for confusion" is sometimes you will have specifically web-browser as a translation, and other times you end up with specifically file-browser. Thus removing the desired(?) ambiguity in languages that don't have it.

Reply:
This context should be clarified through weblate.

Reply:
> and there should be any room for confusion

He meant "shouldn't"

Reply:
@opusforlife2 I wasn't 100% sure myself, and I translated it from English.
"string name="open_in_browser" is of no help, but at-least that has to change for it have any hopes of landing on web-.

Reply:
Sorry, did you mean to address Tobi here? I have no idea about any of this.

Reply:
Looks like this now:
https://hosted.weblate.org/translate/newpipe/strings/en/?checksum=7f6f1617954bdd58

Reply:
I posted a comment to notify all translators, too. @comradekingu Are you okay with that? If yes, I'd suggest to  revert your change back to "browser" and merge this pr.

Reply:
@TobiGr Done. If you can add https://hosted.weblate.org/user/kingu/ as admin in https://hosted.weblate.org/access/newpipe/ I can add source comments and upload screenshots so that no confusion arises.

Reply:
Thank you for the offer. Done.
Btw. do you know, if i can restrict your permissions? I can only choose admin and review. But [according to the docs](https://docs.weblate.org/en/latest/admin/access.html#groups), there should be more groups.

Reply:
@TobiGr I think those are available as top level (Weblate) admins to do (on Hosted), so only for self-hosted instances. There is "review" access, but I don't think there are any more rights attached to it. I got admin access for the ones in https://hosted.weblate.org/user/kingu/#managed
Can get the setup working a bit better some time later, right now there isn't fastlane support and a few other things could be better.